lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -171,7 +177,7 @@ def log.trace?
       # Caches pod/namespace UID tuples for a given container UID.
       @id_cache = LruRedux::TTL::ThreadSafeCache.new(@cache_size, @cache_ttl)
 
-      # Use the container UID as the key to fetch a hash containing pod metadata
+      # Use the pod UID as the key to fetch a hash containing pod metadata
```

Comment:
@richm Is this correct?  The info we infer from the file name is not pod UUID as generated by kubernetes but the container UUID as generated by docker is it not?